.github/workflows/ccpp.yml

```
@@ -121,14 +103,98 @@ jobs:
       shell: bash
 
-    - uses: actions/upload-artifact@v3
+    - uses: actions/upload-artifact@v4
       if: matrix.name == 'windows-msvc'
       with:
-        name: 'assimp-bins-${{ matrix.name }}-${{ github.sha }}'
-        path: build/bin
+        name: 'assimp-bins-${{ matrix.name }}'
+        path: build/bin/assimp*.exe
+        
+    - uses: marvinpinto/action-automatic-releases@latest
```

actionlint (1.7.7)

the runner of "marvinpinto/action-automatic-releases@latest" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)